include/lbann/callbacks/export_onnx.hpp

```diff
@@ -64,19 +70,16 @@ class export_onnx : public callback_base {
   /** @brief Return name of callback */
   std::string name() const override { return "export_onnx"; }
 
-  /* @brief gather model info */
-  void on_setup_end(model* m) override;
-
   /* @brief gather graph/layer info */
-  void on_train_begin(model* m) override;
+  void on_train_end(model* m) override;
 
 private:
 
-  /* @brief option to print onnx debug string */
-  bool m_print_debug_string;
+  /* @brief name of output file. Default = lbann.onnx */
+  std::string m_output_filename;
 
-  /* @brief name of output file. Default = lbann_output.onnx */
-  std::string m_output_file;
+  /* @brief option to print onnx debug file. Default = none */
+  std::string m_debug_string_filename;
 
   /* @brief onnx ModelProto object */
   onnx::ModelProto mp_;
```

Comment:
Remove this. Just create a new `onnx::ModelProto` in the scope of `on_train_end()`.